src/main/java/forestry/apiculture/EventHandlerApiculture.java

```diff
@@ -34,19 +34,43 @@ public void addFrameTooltip(ItemTooltipEvent event) {
                     tooltip.add(StringUtil.localize("frame.tooltip.durability") + getDurabilityFormatted(durability));
 
                     float territory = modifier.getTerritoryModifier(null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.territory") + getColorFormatted(territory));
-
+                    if (territory != 1.0F) {
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.territory")
+                                        + getModifierFormatted(territory, false, false));
+                    }
                     float mutation = modifier.getMutationModifier(null, null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.mutationRate") + getColorFormatted(mutation));
-
+                    if (mutation != 1.0F) {
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.mutationRate")
+                                        + getModifierFormatted(mutation, false, false));
+                    }
                     float lifespan = modifier.getLifespanModifier(null, null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.lifespan") + getColorFormatted(lifespan));
-
-                    float production = modifier.getProductionModifier(null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.production") + getColorFormatted(production, true));
-
+                    if (lifespan != 1.0F) {
+                        // lower lifespan is better
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.lifespan")
+                                        + getModifierFormatted(lifespan, false, true));
+                    }
+                    float production = modifier.getProductionModifier(null, 0.0F);
+                    if (production != 0.0F) {
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.production")
+                                        + getModifierFormatted(production, true, false));
+                    }
+                    float flowering = modifier.getProductionModifier(null, 1.0F);
```

Comment:
Wrong modifier
```suggestion
                    float flowering = modifier.getFloweringModifier(null, 1.0F);
```

Reply:
Fixed in https://github.com/GTNewHorizons/ForestryMC/pull/55/commits/a868679d8ec93f2201ba65fea73e1c2f303f34bd